### PR TITLE
Android client version bump

### PR DESCRIPTION
**Pull request by beersheba**
- Update Android library build script for Mac
- Correct android client libraries version

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test

File: scripts/android-build-common.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ if [ -z $NDK_TARGET ]; then
 fi
 
 if [ -z $CMAKE_PROGRAM ]; then
-  	CMAKE_PROGRAM=$(find $ANDROID_SDK/cmake -name cmake -type f -executable -print -quit)
+  	CMAKE_PROGRAM=$(find $ANDROID_SDK/cmake -name cmake -type f -print -quit)
```
Comment (akallabeth) [inline scripts/android-build-common.sh:15]: sorry, this breaks the sdk/ndk stuff on our build machines :/

Comment (beersheba) [reply scripts/android-build-common.sh:15]:
```
Command /opt/android-sdk-linux/cmake/3.10.2.4988404/share/cmake-3.10/completions/cmake not found. Install and add it to the PATH
```
@akallabeth did my change caused it?

Comment (akallabeth) [reply scripts/android-build-common.sh:15]: yes. it finds the first file matching `cmake` but that is not necessarily an `executable`

Comment (beersheba) [reply scripts/android-build-common.sh]: reverted. can you review it again?

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**